--- qmail-scanner-queue.template	2004-09-24 02:30:16.000000000 +0300
+++ n/qmail-scanner-queue.template	2004-09-24 02:28:37.000000000 +0300
@@ -307,10 +307,10 @@
 }
 
 chdir($scandir);
-umask(0077);
+umask(0027);
 
 if (! -d "$scandir/tmp") {
-  mkdir("$scandir/tmp",0700) || &error_condition("cannot create $scandir/tmp - $!");
+  mkdir("$scandir/tmp",0750) || &error_condition("cannot create $scandir/tmp - $!");
 }
 
 my ($quarantine_event,$quarantine_event_tmp)=0;
@@ -516,7 +516,7 @@
   select(STDIN); $|=1;
   
   &debug("w_c: mkdir $ENV{'TMPDIR'}");
-  mkdir("$ENV{'TMPDIR'}",0700)||&error_condition("$ENV{'TMPDIR'} exists - try again later...");
+  mkdir("$ENV{'TMPDIR'}",0750)||&error_condition("$ENV{'TMPDIR'} exists - try again later...");
   chdir("$ENV{'TMPDIR'}")||&error_condition("cannot chdir to $ENV{'TMPDIR'}/");
   if (-f "$scandir/$wmaildir/tmp/$file_id" || -f "$scandir/$wmaildir/new/$file_id") {
     &error_condition("$file_id exists, try again later");
