--- ../qmail-scanner-queue.template	2004-12-10 14:24:52.000000000 +0200
+++ b/qmail-scanner-queue.template	2004-12-10 14:25:44.000000000 +0200
@@ -92,11 +92,11 @@
 
 
 #From: line  information used when making reports
-my $V_FROM='USERNAME@MAILDOMAIN';
+our $V_FROM='USERNAME@MAILDOMAIN';
 my $V_FROMNAME='System Anti-Virus Administrator';
 
 # Address carbon-copied on any virus reports
-my $QUARANTINE_CC='USERNAME@MAILDOMAIN';
+our $QUARANTINE_CC='USERNAME@MAILDOMAIN';
 
 #Array of local domains that are checked against for
 #deciding whether or not to send recipient alerts to
@@ -107,10 +107,10 @@
 
 
 #Array of virus scanners used must point to subroutines
-my @scanner_array=(SCANNER_ARRAY);
+our @scanner_array=(SCANNER_ARRAY);
 
 #Addresses that should be alerted of any quarantined Email
-my $NOTIFY_ADDRS='NOTIFY_ADDRESSES';
+our $NOTIFY_ADDRS='NOTIFY_ADDRESSES';
 
 #Try to fix bad MIME messages before passing to MIME unpacker
 my $BAD_MIME_CHECKS='FIX_MIME';
@@ -259,7 +259,7 @@
 
 
 #Want debugging? Enable this and read $scandir/qmail-queue.log
-my $DEBUG='DEBUG_LEVEL';
+our $DEBUG='DEBUG_LEVEL';
 
 my @uufile_list = ();
 my @attachment_list = ();
